### PR TITLE
Clarify membership is not required

### PR DESCRIPTION
**Pull request by Beanow**
Part of several improvements we could make to explain membership better. See #64.

This PR tries to address that membership could be misunderstood as a requirement for participation.

### CODE REVIEW

**Comment by Beanow** [comment]
cc @nothingismagick @vsoch @themightychris

File: MEMBERSHIP.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
 # Membership in the SFOSC
 
 Welcome! Anyone who is interested in contributing to the SFOSC in any way is welcome
-to become a member. At the moment, the only privilege membership brings is being able
+to become a member.
+
+Membership is **not a requirement** for participating in the community.
+Members may be consulted about governance of the community. For example through votes.
+The [CONTRIBUTING GUIDELINES](https://github.com/sfosc/sfosc/blob/master/CONTRIBUTING.md)
+explains several ways you can contribute regardless of membership.
```
Comment (nothingismagick) [inline MEMBERSHIP.md]: "Guidelines explain" not "guidelines explains"

File: SOCIAL_CONTRACT.md [hunk 2]
```diff
@@ -50,6 +50,11 @@ in the community requires following the Code of Conduct.
 
 ### Membership
 
+Membership is **not a requirement** for participating in the community.
+Members may be consulted about governance of the community. For example through votes.
+The [CONTRIBUTING GUIDELINES](https://github.com/sfosc/sfosc/blob/master/CONTRIBUTING.md)
+explains several ways you can contribute regardless of membership.
```
Comment (nothingismagick) [inline SOCIAL_CONTRACT.md]: same

File: MEMBERSHIP.md (hunk 1 above)
Comment (Beanow) [reply MEMBERSHIP.md]: Good point, fixed this.

**Comment by vsoch** [comment]
This looks good to me as well! The one thought that popped to mind while reading is that participation "in any fashion" and "in any way" might not be necessary to say - calling someone a participant suggests that they are doing exactly this! Would it make sense to tweak a bit, so it possibly sounds stronger? E.g.,

> Anyone who is interested in contributing to the SFOSC is welcome to become a member

And then for the second.

> Membership in the community is broad based, available to any participant. 

If it's not clear what participation means, the reader could reference contributing.md. It sort of should be, since GitHub / issues / PRs / talking to people is a fairly well known thing.

**Comment by Beanow** [comment]
@vsoch I see what you mean. Though I think it would be better to stress this point too much than not enough.

Using "available to any participant" could in the worst case confuse people to think, am I missing another term "participant" here? Using the verb + "in any fashion" I think avoids that.

The "in any fashion", part I think is also meant to emphasize developers / people on GitHub don't get privileged treatment. Probably not very applicable to SFOSC currently. But may have gone in there so it can eventually double as a template (for https://sfosc.org/social-contracts/) and future proofing. Especially when you're moving outside GitHub, for example with a community translation platform, I think it would be good to stress it's meant to include contributions in the broadest sense.

**Comment by Beanow** [comment]
@vsoch I'll assume you didn't mean your comment as blocking and go ahead merging this.
Always happy to revisit if you disagree :smile:

**Pull request merged**